### ISSUE TITLE
cl31 bad error string

### ISSUE THREAD

**Issue by madrouin** [ACTRIS-CCRES/raw2l1#16]
In GitLab by @aurelien.faucheux on Sep 28, 2020, 21:49

Bad error strings (f, 6, a, 2, e) or error string 0 with a lot of messages (sometimes all) in alarm infos on those dates:

Dates / error string / alarm infos
2020-01-24 09:41:34 / f / 009bffbfcffd  
2020-01-29 14:43:53 / 0 / fc72ffc78fff  
2020-01-31 14:57:18.000000256 / 0 / ffdc0ffeaf00  
2020-02-07 15:19:32.999999744 / 0 / ffc9bffda1ff  
2020-02-26 14:36:21 / f / fe1bfffb1ffe  
2020-02-26 14:36:51 / f / 4700113fffd9  
2020-02-28 10:19:47 / f / a0006b000660  
2020-03-11 09:06:14 / 6 / bffee9fffb4f  
2020-03-16 08:55:33.000000256 / c / 173001770019  
2020-04-01 12:13:27 / 0 / fffebffbdb00  
2020-05-07 11:09:10 / 0 / 0c4fffc80008  
2020-05-18 10:56:17 / 2 / ffff90003c00  
2020-05-25 11:55:32 / a / 7d00028fff99  
2020-06-08 14:29:03 / 0 / 1afff61fff55  
2020-06-19 06:29:40.999999744 / f / 186001f7ffff  
2020-06-26 08:20:56 / f / fac00014fffb  
2020-07-01 12:28:44 / 0 / 002bf000e900  
2020-07-08 13:47:29 / 2 / 00700098fffe  
2020-07-13 08:57:19 / f / 20fff67ffe78  
2020-07-15 08:03:15.999999744 / 3 / ffb0002a0001  
2020-07-24 10:06:57.000000256 / 0 / fdd6ffddb001  
2020-07-29 08:19:46 / e / 3000c2ffe8af

**Comment by madrouin** [comment]
In GitLab by @aurelien.faucheux on Sep 28, 2020, 21:49

changed due date to December 31, 2020

**Comment by madrouin** [comment]
In GitLab by @aurelien.faucheux on Dec 30, 2020, 10:07

changed due date to January 04, 2022

**Comment by madrouin** [comment]
Can you confirm you only detect these problem on files from MeteoFrance CL31 ?

If it is the case, I think the easiest way is to use [the script made by Christophe](https://gitlab.in2p3.fr/-/snippets/622) to check data massages from Vaisala CL31 et CL51 and only keep the one fallowing the right format before running raw2l1.

**Comment by madrouin** [comment]
In GitLab by @aurelien.faucheux on Sep 27, 2021, 16:58

Do you mean CL31 with RecProfil acquisition data? If so, I think I did not test data before 2020.

**Comment by madrouin** [comment]
No I mean data from airports

**Comment by madrouin** [comment]
In GitLab by @aurelien.faucheux on Sep 27, 2021, 19:27

It's the CL31 from SIRTA. I did not check any others I think. I can try to find the script I used to get those date to check other CL31.

**Comment by madrouin** [comment]
In GitLab by @aurelien.faucheux on Oct 5, 2021, 14:45

I checked for orly, roissy, P7 and SIRTA any character that is not b'0', b'A' or b'W' in all files on bdd. There is no problem in 2021 for the P7, airports and Z1. For the files before :    
For SIRTA the dates are in the attached files [cl31-err_2010-2020.csv](https://my-gitlab-bucket.s3.amazonaws.com/670670512/4d0ea2d3274f46579c252ecb177ceceda456bf69025acf3ba874b068a08d5e7d/cl31-err_2010-2020.csv)  
For P7 between I found those dates :  
2017-02-01 10:41:35.000000000 	b''  
2017-03-10 09:07:40.000000000 	b'f'  
2018-05-23 04:06:33.999999744 	b''  
2018-05-25 21:06:23.000000000 	b''  
2018-09-04 06:43:09.000000256 	b'\x80'  
2018-09-04 06:43:39.000000000 	b'?'  
2018-09-04 06:51:08.999999744 	b'\x80'  
2018-09-04 07:16:39.000000000 	b''  
2018-09-04 07:18:39.000000256 	b''  
2018-11-05 12:42:08.000000000 	b''  
2018-12-19 04:10:41.000000000 	b'f'  

For Roissy see file attached [cl31-roissy-err_2015-2020.csv](https://my-gitlab-bucket.s3.amazonaws.com/670670512/3c64bcd52561d569e5aa0eee2a1038d40820aa5cf850c3597d97df7da0f1afd9/cl31-roissy-err_2015-2020.csv)  
Did not find any character in orly files.

I don't really understand and don't remember what I meant about the 0 string errors
